baldrick/plugins/github_pull_requests_base_branch.py

```diff
@@ -0,0 +1,22 @@
+from baldrick.plugins.github_pull_requests import pull_request_handler
+
+
+# Or do we want to check every push event?
```

Comment:
Nope, the opening one is enough. Sometimes it's legit to open a PR against non master.